test/nodes/utils.py

```diff
@@ -134,7 +134,7 @@ def reset(self, initial_state: Optional[Dict[str, Any]] = None):
             self.i = 0
             self.num_resets += 1
 
-    def next(self) -> Iterator[Dict[str, int]]:
+    def next(self) -> Dict[str, int]:
```

Comment:
Linter was complaining on mismatching types.

Reply:
Thank you linter